.expeditor/publish-release-notes.sh

```diff
@@ -30,6 +30,15 @@ pushd ./automate.wiki
 ## Backward Incompatibilities
 -
 
+## Chef Product Versions
+
+This release uses:
+- Chef Habitat version
+- Chef Habitat Builder version
+- Chef Infra Client version
```

Comment:
infra client version shouldn't be included.

inspec, builder, habitat, and infra server make sense to me

@chef/automate-deployment-team whaddyall think?